### PR TITLE
Fhv encounters

### PR DESCRIPTION
**Pull request by DaveSea79**
I hope I have done this correctly now. Fingers crossed!

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -2904,6 +2904,106 @@
...
+    "faction_code": "mythos",
+    "flavor": "Half the majestic creature's flesh has been replaced with vascular, bark-like tissue.",
+    "health": 5,
+    "illustrator": "Peter Polach",
+    "name": "Cochleal Stag",
+    "pack_code": "fhvc",
+    "position": 234,
+    "quantity": 1,
+    "text": "Elusive. <b>(Night [night])</b> Hunter.\nEach investigator at Cochleal Stag's location and each connecting location gets -2 to all skills during the mythos phase.",
+    "traits": "Creature. Flora. Mutated.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "traits": "Creature. Monster. Flora. Mutated.",
```

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -2904,6 +2904,106 @@
...
+    "code": "10735",
+    "encounter_code": "the_forest",
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "illustrator": "Rob Laskey",
+    "name": "Bloom",
+    "pack_code": "fhvc",
+    "position": 235,
+    "quantity": 2,
+    "text": "<b>Revelation</b> - Choose the nearest [[Flora]] enemy and heal all damage from it (if that enemy is [[Elite]], heal 2[per_investigator] damage from it instead). If no damage was healed by this effect, discard cards from the top of the encounter deck until a [[Flora]] card is discarded and draw it.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "<b>Revelation</b> - Choose the nearest [[Flora]] enemy and heal all damage from it (if that enemy is [[Elite]], heal 2 [per_investigator] damage from it instead). If no damage was healed by this effect, discard cards from the top of the encounter deck until a [[Flora]] card is discarded and draw it.",
```

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -2904,6 +2904,106 @@
...
+    "text": "<b>Revelation</b> - Attach Wall of Thorns to the nearest location without Wall of Thorns attached.\n<b>(Day [day]) Forced</b> - When you move into attached location: Take 2 damage. Discard Wall of Thorns.\n<b>(Night [night]) Forced</b> - When you move into or out of attached location: Take 1 damage.",
+    "traits": "Hazard. Flora.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10737",
+    "encounter_code": "the_forest",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "flavor": "\"...even the herbivores seem to have developed an unusual ability to plan, coordinate, and hunt.\"\n - Dr. Rosa Marquez, Miskatonic Survey, 1926",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "flavor": "\"...even the herbivores seem to have developed an unusual ability to plan, coordinate, and hunt.\"\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: We pop a wee underline under italicised titles in flavor text, just to show they are a book title.

**Review by drawntotheflame — COMMENTED** [review]
Great job @DaveSea79 - just a few tiny tweaks for you. Thanks!

**Comment by DaveSea79** [comment]
Thanks @drawntotheflame - all committed.
@zzorba I'm assuming I just leave this open for you to incorporate, but let me know if I need to do anything further.

**Pull request merged**